lib/ansible/plugins/filter/ipaddr.py

```diff
@@ -29,6 +29,7 @@
 except ImportError:
     # in this case, we'll make the filters return error messages (see bottom)
     netaddr = None
+    NETADDR_ERRORS = ()
```

Comment:
I put it here like netaddr don't know if it's more logic

Reply:
Yes, you can do it like this.